app/models/submissions/questions_sub.rb

```diff
@@ -4,6 +4,22 @@
 class QuestionsSub < Submission
   include SubmissionsHelper
   validate :check_time_taken
+
+
+  def grade_question!(who_graded, grade_id, message, question, score)
```

Comment:
The naming convention in Ruby is that names ending in `!` either succeed or error.  However, `comment.update` returns a boolean for its success or failure, which you ignore and then just return the comment...